packages/flame/lib/src/text.dart

```diff
@@ -46,6 +46,19 @@ abstract class TextRenderer<T extends BaseTextConfig> {
       measureTextHeight(text),
     );
   }
+
+  /// Creates a new instance of this painter but transforming the [config]
+  /// object via the provided lambda.
+  TextRenderer<T> copyWith(T Function(T) transform);
+
+  /// Given a generic type [T], creates a default renderer of that type.
+  static T createDefault<T extends TextRenderer>() {
```

Comment:
I am not sure about this, this method will make it impossible for people to create their own implementation right? All implementation will need to be provided by Flame

Reply:
yeah that makes sense